apps/etl-func/src/adapters/pdv-tokenizer/pdv-tokenizer-client.ts

```diff
@@ -0,0 +1,57 @@
+import { TokenizerClient } from "@/domain/interfaces/tokenizer.js";
+
+import { piiResourceSchema } from "./pii-resource.js";
+import { problemSchema } from "./problem.js";
+import { tokenResourceSchema } from "./token-resource.js";
+
+const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+
+export default class PDVTokenizerClient implements TokenizerClient {
+  #apiKey: string;
+  #basePath = "tokenizer/v1";
```

Comment:
no need to have `basePath` as property, since is not used in other methods (and it be part of the base url)